include/sender_cadmium.hpp

```diff
@@ -70,10 +70,11 @@ class Sender {
             
 	    // ports definition
         using input_ports=std::tuple<typename defs::control_in,
-		                             typename defs::ack_in>;
+            typename defs::ack_in>;
         using output_ports=std::tuple<typename defs::packet_sent_out,
-		                             typename defs::ack_received_out,
-	                                     typename defs::data_out>;
+	    typename defs::ack_received_out,
+	        typename defs::data_out>;
```

Comment:
typename defs::data_out>; --> it can be in the same line with typename defs::ack_received_out. 
It will satisfy the < 80 character requirement

Reply:
Resolved